command/commands/token.py

```diff
@@ -30,11 +35,13 @@ def __init__(self, db_facade, config):
         self.expiry = config.expiry
         self.signing_key = config.signing_key
 
-    def handle(self, _command, user_id):
+    def handle(self,
+               _command: str,
+               user_id: str) -> ResponseTuple:
         """Handle request for token."""
         logging.debug("Handling token command")
         try:
-            user = self.facade.retrieve(User, user_id)
+            user = cast(User, self.facade.retrieve(User, user_id))
```

Comment:
I don't know much about how typing works, but would it be possible for the facade to cast the object it retrieves from the database before returning it? Saves from having to cast in every instance the facade returns something.